R/utilities.R

```diff
@@ -2731,6 +2906,50 @@ collate_lymphgen = function(sample_table,
 }
 
 
+#' Expand a sample_table (meta data) horizontally with different flavours of lymphgen data.
+#'
+#' @param sample_table Input data frame with sample IDs in the firs column.
+#'
+#' @return
+#' @export
+#' @import tidyverse
+#'
+#' @examples
+#' metadata = get_gambl_metadata()
+#' colalted_meta = collate_lymphgen(sample_table = metadata)
+#' 
+collate_lymphgen = function(sample_table){
+  
+  #setup paths
+  repo_base = config::get("repo_base") #get repo base
+  flavours = config::get("results_merged_wildcards")$lymphgen_template #get all flavours
+  flavours = str_split(flavours, pattern = ",") #seperate each flavour on ","
+  flavours = unlist(flavours) #unlist flavours
+  lymphgen_template = config::get("results_versioned")$lymphgen_template #get tamplate for lymphgen file path
+  lymphgen_path = paste0(repo_base, lymphgen_template) #combine with repo_base path
+  lymphgen_full_paths = glue::glue(lymphgen_path, flavour = flavours) #substitute {flavour} wildcard in template with actual flavours
```

Comment:
Does this actually work? I thought it would just substitute in a vector from namespace so the part specifying flavour=flavours isn't necessary if you instead had your vector named "flavour".